data.tf

```diff
@@ -1,15 +1,3 @@
-# Base AMI is found by ID - this is to prevent issues where launch configurations
```

Comment:
I tried, but that comment seems to try and warn about something, but I can't figure it out.